saml2/saml2-service-provider/core/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java

```diff
@@ -134,7 +134,7 @@ public String getRegistrationId() {
 
 	/**
 	 * Get the relying party's <a href=
-	 * "https://wiki.shibboleth.net/confluence/display/CONCEPT/EntityNaming">EntityID</a>.
+	 * "https://www.oasis-open.org/committees/download.php/51890/SAML%20MD%20simplified%20overview.pdf#2.9%20EntityDescriptor">EntityID</a>.
```

Comment:
Will you please update the copyright header in this file to include the current year?

Reply:
Sure, thanks for pointing this out